Makefile.dep

```diff
@@ -105,6 +105,13 @@ FEATURES_OPTIONAL += no_idle_thread
 ifneq (,$(filter libstdcpp,$(FEATURES_USED)))
   # Also use C++ if libstdc++ is used
   FEATURES_REQUIRED += cpp
+  USEMODULE += cpp
+  USEMODULE += libstdcpp
+endif
+
+ifneq (,$(filter cpp,$(FEATURES_USED)))
+  # Also use C++ if libstdc++ is used
+  USEMODULE += cpp
```

Comment:
I think it makes sense to have `cpp` and `libstdcpp` as pseudomodules which depend on the respective features, but in this case perhaps we want to modify the makefiles to include the module. So instead of `FEATURES_REQUIRED += cpp` a module would `USEMODULE += cpp`. Then, here we would:

```makefile
ifneq (,$(filter libstdcpp,$(USEMODULE)))
  FEATURES_REQUIRED += libstdcpp
  USEMODULE += cpp
endif

ifneq (,$(filter cpp,$(USEMODULE)))
  FEATURES_REQUIRED += cpp
endif
```

Reply:
@fjmolinas do you see any reason not to add these as pseudomodules?

Reply:
I will wait for a conclusion.

Reply:
I guess we could just leave this to a follow-up PR.